docs/content/docs/data-structures/map.md

```diff
@@ -1,10 +1,449 @@
+```
+
+Now, anytime you are looking to load the `newSharedMap` back at a later time, all you have to do is load it back from the stored handle.
+
+```javascript
+const newSharedMapHandle = initialMap.get("newSharedMapKey");
+const newSharedMap = await newSharedMapHandle.get();
+```
+
+{{< callout note >}}
+
+Loading any DDS from its handle is an asynchronous operation. Whereas objects defined in `initialObjects` will be available synchronously once the container is provided, you will need to use an `await` or chain Promises on the `handle.get()` call.
+
+{{< /callout >}}
+
+#### Nested shared objects example
+
+Let's take another look at the example from the **Storing objects** section above to see how it can be updated to use nested `SharedMaps`. This will introduce some data hierarchy and avoid having a flat map that has both the tasks and their fields themselves on the same level. To do so, let's start from the earlier model but with two tasks now.
+
+```json
+{
+    "task1": {
+        "title": "Awesome Task",
+        "description": "Doing the most awesome things",
+        "assignedTo": {
+            "name": "Joe Schmo",
+            "email": "joeschmo@email.com",
+            "address": "1234 Fluid Way"
+        }
+    },
+    "task2": {
+        "title": "Even More Awesome Task",
+        "description": "Doing even more awesome things",
+        "assignedTo": {
+            "name": "Jane Doe",
+            "email": "janedoe@email.com",
+            "address": "5678 Framework Street"
+        }
+    }
+}
+```
+Here, you can have each task itself be its own `SharedMap` and have a parent `SharedMap` hold all of the handles to each task.
+
+Then, the initial map would look like:
+```json
+{
+    "task1": task1MapHandle,
+    "task2": task2MapHandle
+}
+```
+And the `task1` map would have:
+```json
+{
+    "title": "Awesome Task",
+    "description": "Doing the most awesome things",
+    "assignedTo": {
+        "name": "Joe Schmo",
+        "email": "joeschmo@email.com",
+        "address": "1234 Fluid Way"
+    }
+}
+```
+And the `task2` map would have:
+```json
+{
+    "title": "Even More Awesome Task",
+    "description": "Doing even more awesome things",
+    "assignedTo": {
+        "name": "Jane Doe",
+        "email": "janedoe@email.com",
+        "address": "5678 Framework Street"
+    }
+}
+```
+With this nested map structure, you are able to both ensure that each field that will be collaboratively edited is stored separately and that you have a hierarchy in how you store the data in your `SharedMap` that reflects the app's data model.
+
+Whenever a new task is created, you can call `container.create` to get a new `SharedMap` instance and store its handle into the `initialMap` that is provided as an `initialObject`. Since each additional task map would need its own unique key, you can use a random string generator, such as [uuid](https://www.npmjs.com/package/uuid), to create a new one each time and avoid any clashes.
+
```

Comment:
```suggestion
With this nested map structure, you can both ensure that each field that will be simultaneously edited is stored separately and that you have a hierarchy in how you store the data in your `SharedMap` that reflects the app's data model.

Whenever a new task is created, you can call `container.create` to create a new `SharedMap` instance and store its handle in the `initialMap` that is provided as an `initialObject`. Since each additional task map will need its own unique key, you can use a random string generator, such as [uuid](https://www.npmjs.com/package/uuid), to create a new one each time and avoid ID conflicts.

```